Makefile

```diff
@@ -1,31 +1,42 @@
 install:
+	@export PR=https://api.github.com/repos/$$TRAVIS_REPO_SLUG/pulls/$$TRAVIS_PULL_REQUEST; \
+	export BRANCH=$$(if [ "$$TRAVIS_PULL_REQUEST" == "false" ]; then echo $$TRAVIS_BRANCH; else echo `curl -s $$PR | jq -r .head.ref`; fi); \
+	echo "TRAVIS_BRANCH=$$TRAVIS_BRANCH, PR=$$PR, BRANCH=$$BRANCH"; \
+	pattern="refactor_"; \
+	if [[ $$BRANCH =~ $$pattern ]]; then \
+		repo=github.com/dedis/cosi; \
+		go get $$repo; \
+		cd $$GOPATH/src/$$repo; \
+		git checkout -f $BRANCH; \
+	fi;\
+	cd $$GOPATH/src/github.com/dedis/cothority; \
 	go get -t ./...
 
 test_fmt:
 	@echo Checking correct formatting of files
 	@{ \
-	files=$$( go fmt ./... ); \
-	if [ -n "$$files" ]; then \
+		files=$$( go fmt ./... ); \
+		if [ -n "$$files" ]; then \
 		echo "Files not properly formatted: $$files"; \
 		exit 1; \
-	fi; \
-	if ! go vet ./...; then \
+		fi; \
+		if ! go vet ./...; then \
 		exit 1; \
-	fi \
-	}
+		fi \
+		}
```

Comment:
Were this new tabs intentionally created? If yes, why is the closing bracket on a different level than the opening one?